security/.htaccess

```diff
@@ -5,30 +5,26 @@ RewriteEngine on
 RewriteRule ^$ https://w3c-ccg.github.io/security-vocab/ [R=302,L]
 RewriteRule ^v1$ https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld [R=302,L]
 RewriteRule ^v2$ https://w3c-ccg.github.io/security-vocab/contexts/security-v2.jsonld [R=302,L]
+
+# TODO: make v3 "stable"
+
 RewriteRule ^v3-unstable$ https://w3c-ccg.github.io/security-vocab/contexts/security-v3-unstable.jsonld [R=302,L]
 
-# https://w3id.org/security/jws
+# TODO: remove "suites" not rooted under  "suites"
```

Comment:
@tplooker @dlongley @dmitrizagidulin 

we probably ought to just bite the bullet and kill these older URIs asap... the longer we wait, the madder everyone will be.

Reply:
Yeah that's a good point...

Reply:
Related: https://github.com/w3c-ccg/security-vocab/issues/108